.eslintrc.js

```diff
@@ -43,6 +43,10 @@ module.exports = {
       }
     }],
     'one-var': ['error', 'never'],
+    'import/no-extraneous-dependencies': ['error', {
+      /* the following folders can import devDependencies */
+      devDependencies: ['test/**', 'tests/**', 'src/debug/**', 'examples/**'],
```

Comment:
> Then, AFAIK, dist/ would be included in the package because it's not ignored (and it is generated by npm build during any "NPM release process"

Oh right I didn't know `npm install` was done automatically for npm release. But I still think we should commit it when doing a git/github release, because it supports an npm installation directly from our github repo. Are these workflows compatible?